resources/install/protected/update_NEXT.sql

```diff
@@ -0,0 +1,79 @@
+DROP TABLE IF EXISTS `AuthorizedSite`;
+CREATE TABLE  `AuthorizedSite` (
+  `authorizedSiteID` int(11) NOT NULL auto_increment,
+  `shortName` varchar(45) default NULL,
+  PRIMARY KEY  (`authorizedSiteID`)
+) ENGINE=MyISAM AUTO_INCREMENT=1 DEFAULT CHARSET=utf8;
+
+DROP TABLE IF EXISTS `Resource`;
```

Comment:
Wow, careful!  Why are we dropping and recreating the Resource table?  This will lose all data in the table.